src/components/Tokens/TokenRowWallets.tsx

```diff
@@ -212,6 +217,8 @@ export const TokenRowWallets = component$<TokenRowWalletsProps>(
       }
     });
 
+    if (!Number(currentBalanceOfToken.value)) return <></>;
```

Comment:
check both positive cases currentbalanceoftoken.value || latesttokenprice.value, use ternary operator

Reply:
done. is it effect we wanted? i guess it is
![image](https://github.com/emeth-defi-automation/portfolio-management-platform/assets/93248555/0e72bdb2-022b-426f-b33c-e676197961a0)

Reply:
good job!